client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -144,47 +186,76 @@ class ActivityLogDay extends Component {
 			requestRestore,
 			rewindConfirmDialog,
 			siteId,
+			tsEndOfSiteDay,
 		} = this.props;
 
-		const hasLogs = ! isEmpty( logs );
+		const hasLogs = ! isEmpty( logs ),
+			rewindHere = this.state.rewindHere,
+			dayExpanded = this.state.dayExpanded ? true : rewindHere;
```

Comment:
note that according to our style guides we're putting these on separate lines. the `vars` get grouped but not the `let`s and `const`s

```js
const hasLogs = ! isEmpty( logs );
const { rewindHere } = this.state;
const dayExpanded = this.state.dayExpected ? true : rewindHere;
```

Reply:
Thanks, I've updated this. I wasn't aware of this guideline and will keep it mind in the future.